src/PKSim.Core/Services/SimulationConfigurationValidator.cs

```diff
@@ -47,9 +49,41 @@ public void ValidateConfigurationFor(Simulation simulation)
          if (!speciesPopulation.IsBodySurfaceAreaDependent && schemaItems.Any(x => x.DoseIsPerBodySurfaceArea()))
             throw new InvalidSimulationConfigurationException(PKSimConstants.Error.DosePerBodySurfaceAreaProtocolCannotBeUsedWithSpeciesPopulation(speciesPopulation.DisplayName));
 
-         if(speciesPopulation.IsNamed(CoreConstants.Population.PREGNANT))
+         if (speciesPopulation.IsNamed(CoreConstants.Population.PREGNANT))
             throw new InvalidSimulationConfigurationException(PKSimConstants.Error.PregnantPopulationCanOnlyBeUsedWithMoBiModel(speciesPopulation.DisplayName));
 
+         var administeredCompoundWithSuperSaturationEnabled = compounds.Where(x => x.SupersaturationEnabled).Select(x => new
+         {
+            Compound = x,
+            simulation.CompoundPropertiesFor(x).ProtocolProperties
+         }).Where(x => x.ProtocolProperties.IsAdministered).Select(x => new
+         {
+            x.Compound,
+            x.ProtocolProperties.Protocol,
+            x.ProtocolProperties.FormulationMappings
+         });
+
+         administeredCompoundWithSuperSaturationEnabled.Each(x => validateSupersaturationUsageFor(x.Compound, x.Protocol, x.FormulationMappings));
+      }
+
+      private void validateSupersaturationUsageFor(Compound compound, Protocol protocol, IReadOnlyList<FormulationMapping> formulationMappings)
```

Comment:
This is the code called for a supersaturated compound, the protocol and the formulations used for that protocol and compound